postgresql/manifests/postgres.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/path: /metrics
+    prometheus.io/port: "9187"
+    uw.health.aggregator.enable: "false"
+spec:
+  ports:
+    - name: db
+      port: 5432
+      protocol: TCP
+    - name: exporter
+      protocol: TCP
+      port: 9187
+  selector:
+    app: postgres
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgres
+  labels:
+    app: postgres
+spec:
+  serviceName: postgres
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgres
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/path:   /__/metrics
+        prometheus.io/port:   "8081"
+      labels:
+        app: postgres
+      name: postgres
+    spec:
+      containers:
+      - name: metrics
+        env:
+        - name: DATA_SOURCE_NAME
+          valueFrom:
+            secretKeyRef:
+              name: postgres-secrets
+              key: url
+        resources:
+          limits:
+            memory: 64Mi
+            cpu: 100m
+        ports:
+          - containerPort: 9187
+            protocol: TCP
+        imagePullPolicy: Always
+        image: wrouesnel/postgres_exporter:v0.8.0
```

Comment:
Bikeshed: I've been eyeing 
https://github.com/prometheus-community/postgres_exporter as a replacement for `wrouesnel` one that doesn't seem to be supported anymore.

Reply:
A good shout, I will move over. Looks to be better supported anyway?